guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc

```diff
@@ -29,9 +29,15 @@ endif::[]
 
 .Multi-homed {Project} and {SmartProxy}
 
+ifeval::["{build}" == "satellite"]
 Multi-homed {Project} is not supported.
 
 Multi-homed {SmartProxy} is supported, to implement this, you can configure {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+You can use multi-homed {SmartProxies} by configuring {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
```

Comment:
This implies it's the only way. Would something like this be more appropriate?
```suggestion
One way to use multi-homed {SmartProxies} is by configuring {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
```

Reply:
Sure this works perfect!